src/hangar/backends/selection.py

```diff
@@ -22,45 +22,47 @@
 convention, the first digit of the code can be used to identify the storage
 medium:
 
-    * Lowercase ascii letters & digits [0, 1, 2, 3, 4] -> reserved for
+   *  Lowercase ``ascii_letters`` & digits ``[0, 1, 2, 3, 4]`` -> reserved for
       backends handling data on the local disk.
-    * Uppercase ascii letters & digits [5, 6, 7, 8, 9] -> reserved for
+
+   *  Uppercase ``ascii_letters`` & digits ``[5, 6, 7, 8, 9]`` -> reserved for
       backends referring to data residing on a remote server.
 
 This is not a hard and fast rule though, and can be changed in the future if the
 need arises.
 
 Process & Guarantees
----------------------
+--------------------
 
 In order to maintain backwards compatibility across versions of Hangar into the
 future the following ruleset is specified and MUST BE HONORED:
 
-    * When a new backend is proposed, the contributor(s) provide the class with a
-      canonical name (HDF5_00, TILEDB_01, etc) for developer consumption in the
-      backend. The review team will provide an available two-digit code which all
-      records corresponding to that backend must identify themselves with.
+*  When a new backend is proposed, the contributor(s) provide the class with a
+   canonical name (``HDF5_00``, ``TILEDB_01``, etc) for developer consumption in
+   the backend. The review team will provide an available two-digit code which
+   all records corresponding to that backend must identify themselves with.
+
+*  Once a new backend is accepted, the code assigned to it is PERMANENT &
+   UNCHANGING. The same code cannot be used in the future for other backends.
```

Comment:
As discussed, we will need to make room for evolution here. We need to add a versioning scheme after the identifier, like `TLDB_01_001`, or something of that kind, since issuing a new code for each backend version is going to be unlikely and impractical.